sql/createLikesTable.sql

```diff
@@ -0,0 +1,13 @@
+DROP TABLE IF EXISTS likes;
```

Comment:
We like how you have all of your table creations separated into individual files